src/Entity/ServerInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Drupal\elastic_appsearch\Entity;
+
+use Drupal\Core\Config\Entity\ConfigEntityInterface;
+
+/**
+ * Provides an interface for defining Server entities.
+ */
+interface ServerInterface extends ConfigEntityInterface {
+
+  // Add get/set methods for your configuration properties here.
+
+  public function getDescription();
+
+  public function getHost();
+
+  public function getStatus();
+
+  public function getSecret();
```

Comment:
Please add an appropriate description of each method because that is what going to be inherited by `{@inheritdoc}`